clients/cobol-lsp-vscode-extension/src/services/util/FSUtils.ts

```diff
@@ -91,7 +91,13 @@ export function searchInWorkspace(entityName: string, targetFolders: string[], e
  * @param pathToClear represents the folder to be cleaned.
  */
 export function cleanDirectory(pathToClear: string) {
-    if (fs.existsSync(pathToClear)) {
-        readdirSync(pathToClear).forEach(f => fs.rmSync(path.join(pathToClear, `${f}`), { recursive: true }));
-    }
+    console.log("cleanDirectory", pathToClear);
+    try {
+        if (fs.existsSync(pathToClear)) {
+            readdirSync(pathToClear).forEach(f => fs.rmSync(path.join(pathToClear, `${f}`), { recursive: true, force: true }));
```

Comment:
It is preferable to use [vscode API](https://code.visualstudio.com/api/references/vscode-api#FileSystem) for fs related operations..